tests/integration/test_tables.py

```diff
@@ -416,9 +416,10 @@ def test_parital_row_rowset_query_entity_veiw(self):
     def _test_method(self, schema, resultsAs, partial_changes, expected_results):
         # anything starting with "test" will be considered a test case by nosetests so I had to append '_' to it
 
-        import pandas as pd
-        query_results = syn.tableQuery("SELECT * FROM %s" % utils.id_of(schema), resultsAs=resultsAs)
-        assert_equals(len(query_results), 2)
+        query_results = self._query_with_retry("SELECT * FROM %s" % utils.id_of(schema),
+                                               resultsAs,
+                                               2,
+                                               QUERY_TIMEOUT_SEC)
```

Comment:
there is a chance to do a `None.asDataFrame(...)` in line 424. assert_is_not_none() before that.